config/env.config.js

```diff
@@ -85,15 +91,38 @@ class RedirectPlugin {
                                     return match[1];
                                 }
                                 return null;
-                            };
+                            }
+
+                            function isAllowedUrl(url) {
+                                if (url == null || url.trim().length === 0) {
+                                    return true;
+                                }
+                                
+                                url = decodeURIComponent(url);
```

Comment:
Do we need to decodeURIComponent here, can't the caller be responsible for sending in a decoded URL?

Reply:
Fair enough.  Really, it should be the `getParameterByName` that does it.  I'll make a change.